src/coreComponents/mesh/ObjectManagerBase.cpp

```diff
@@ -842,6 +843,18 @@ void ObjectManagerBase::copyObject( const localIndex source, const localIndex de
   }
 }
 
+void ObjectManagerBase::eraseObject( std::set< localIndex > const & indicesToErase )
+{
+  for( auto & nameToWrapper: wrappers() )
+  {
+    WrapperBase * wrapper = nameToWrapper.second;
+    if( wrapper->sizedFromParent() )
+    {
+      wrapper->erase( indicesToErase );
```

Comment:
am I understanding correctly that you are not really deleting the wrapper but just removing some entries?

Reply:
Yes, that's correct.